### DISCUSSION TITLE
Rename `cmd/rollups-node` to `cmd/crn`?

### DISCUSSION THREAD

**Discussion by gligneul** [cartesi/rollups-node#148] [Ideas]
We could reduce the node binary name to something shorter. `crn` is the acronym of Cartesi Rollups Node. What do you think?

**Comment by renan061** [comment]
`crn` is not bad. But maybe we could have a `cartesi` command and do stuff like `cartesi node run` and `cartesi send-input`. This is in line with what we talked in the meeting about commands.

**Reply by gligneul** [reply]
This `cartesi` command you are referring to would be the command-line interface for the rollups node. I don't like the name `cartesi` because it is too generic; maybe `cartesi-rollups`? (Then I can create an alias to `cr`).

**Comment by gligneul** [comment]
> We could reduce the node binary name to something shorter. `crn` is the acronym of Cartesi Rollups Node. What do you think?

I was thinking a bit more about this issue, and maybe we can leave the commands with long names and suggest aliases in the documentation.

**Comment by renan061** [comment]
I like `cartesi-rollups` and the alias idea.
Besides, everybody tabs the command names with autocomplete.
